teuthology/suite/run.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, args):
         self.timestamp = self.args.timestamp or \
             datetime.now().strftime('%Y-%m-%d_%H:%M:%S')
         self.user = self.args.user or pwd.getpwuid(os.getuid()).pw_name
+        self.flavor = self.args.flavor or "default"
```

Comment:
Is this line (and `self.flavor` member in general) actually needed?  Wouldn't argument parsing always populate `self.args.flavor`?

Reply:
yep, that should be in base config now